swift/WorkflowTesting/Sources/WorkflowRenderTester.swift

```diff
@@ -270,16 +270,14 @@ fileprivate final class RenderTestContext<T: Workflow>: RenderContextType {
         let sink = Sink<Action> { action in
             observer.send(value: AnyWorkflowAction(action))
         }
-        subscribe(signal: signal)
```

Comment:
Because `subscribe` has been removed I moved its logic up into this function and am subscribing directly — i.e. I'm not wrapping the `Signal` here in a worker (it creates test failures due to unexpected workers being encountered).